docs/design_decisions.md

```diff
@@ -29,9 +29,6 @@ _**Ratio**_
 
 In sommige development-omgevingen leveren hoofdletters, spaties of speciale tekens in enumeratie-waarden een probleem op met code-genereren.
 
-### DD1.5 Namen van endpoints, url's en parameters bevatten alleen kleine letters
-Voor de namen van endpoints, url's en parameters worden alleen kleine letters gebruikt.
-
 ### DD1.6 Naamgeving van properties worden beïnvloed door de kardinaliteit
```

Comment:
Ik zou het nummer en de naam van de design desicion wel laten staan en daaronder aangeven dat we die hebben laten vervallen. Nu valt er een gat in de nummering en weten we niet waarom.

Reply:
Heb ik aangepast